src/services/API/sqs.js

```diff
@@ -0,0 +1,51 @@
+import { SQSClient, SendMessageCommand } from "@aws-sdk/client-sqs";
+
+const REGION = "ap-south-1";
+const QUEUEURL = "https://sqs.ap-south-1.amazonaws.com/111766607077/EventQueue";
+const sqsClient = new SQSClient({
+  region: REGION,
+  credentials: {
+    accessKeyId: process.env.VUE_APP_AWS_SQS_ACCESS_KEY,
+    secretAccessKey: process.env.VUE_APP_AWS_SQS_SECRET_ACCESS_KEY,
+  },
+});
+
+async function sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userID, isUserValid, authType) {
```

Comment:
```suggestion
export async function sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userID, isUserValid, authType) {
```
You can export directly from here itself